Index: src/main/java/com/answer/tree/Q590_N_ary_Tree_Postorder_Traversal.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q590_N_ary_Tree_Postorder_Traversal.java b/src/main/java/com/answer/tree/Q590_N_ary_Tree_Postorder_Traversal.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q590_N_ary_Tree_Postorder_Traversal.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q590_N_ary_Tree_Postorder_Traversal {
+}
Index: src/main/java/com/answer/tree/Q1631_Path_With_Minimum_Effort.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q1631_Path_With_Minimum_Effort.java b/src/main/java/com/answer/tree/Q1631_Path_With_Minimum_Effort.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q1631_Path_With_Minimum_Effort.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q1631_Path_With_Minimum_Effort {
+}
Index: src/main/java/com/answer/sorting/Q1051_Height_Checker.java
===================================================================
diff --git a/src/main/java/com/answer/sorting/Q1051_Height_Checker.java b/src/main/java/com/answer/sorting/Q1051_Height_Checker.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/sorting/Q1051_Height_Checker.java	
@@ -0,0 +1,4 @@
+package com.answer.sorting;
+
+public class Q1051_Height_Checker {
+}
Index: src/main/java/com/answer/bit/Q201_Bitwise_AND_of_Numbers_Range.java
===================================================================
diff --git a/src/main/java/com/answer/bit/Q201_Bitwise_AND_of_Numbers_Range.java b/src/main/java/com/answer/bit/Q201_Bitwise_AND_of_Numbers_Range.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/bit/Q201_Bitwise_AND_of_Numbers_Range.java	
@@ -0,0 +1,4 @@
+package com.answer.bit;
+
+public class Q201_Bitwise_AND_of_Numbers_Range {
+}
Index: src/main/java/com/answer/tree/Q250_Count_Univalue_Subtrees.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q250_Count_Univalue_Subtrees.java b/src/main/java/com/answer/tree/Q250_Count_Univalue_Subtrees.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q250_Count_Univalue_Subtrees.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q250_Count_Univalue_Subtrees {
+}
Index: src/main/java/com/answer/bit/Q371_Sum_of_Two_Integers.java
===================================================================
diff --git a/src/main/java/com/answer/bit/Q371_Sum_of_Two_Integers.java b/src/main/java/com/answer/bit/Q371_Sum_of_Two_Integers.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/bit/Q371_Sum_of_Two_Integers.java	
@@ -0,0 +1,4 @@
+package com.answer.bit;
+
+public class Q371_Sum_of_Two_Integers {
+}
Index: src/main/java/com/answer/tree/Q285_Inorder_Successor_in_BST.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q285_Inorder_Successor_in_BST.java b/src/main/java/com/answer/tree/Q285_Inorder_Successor_in_BST.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q285_Inorder_Successor_in_BST.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q285_Inorder_Successor_in_BST {
+}
Index: src/main/java/com/answer/bit/Q504_Base_7.java
===================================================================
diff --git a/src/main/java/com/answer/bit/Q504_Base_7.java b/src/main/java/com/answer/bit/Q504_Base_7.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/bit/Q504_Base_7.java	
@@ -0,0 +1,4 @@
+package com.answer.bit;
+
+public class Q504_Base_7 {
+}
Index: src/main/java/com/answer/graph/Q1136_Parallel_Courses.java
===================================================================
diff --git a/src/main/java/com/answer/graph/Q1136_Parallel_Courses.java b/src/main/java/com/answer/graph/Q1136_Parallel_Courses.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/graph/Q1136_Parallel_Courses.java	
@@ -0,0 +1,4 @@
+package com.answer.graph;
+
+public class Q1136_Parallel_Courses {
+}
Index: src/main/java/com/answer/tree/Q449_Serialize_and_Deserialize_BST.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q449_Serialize_and_Deserialize_BST.java b/src/main/java/com/answer/tree/Q449_Serialize_and_Deserialize_BST.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q449_Serialize_and_Deserialize_BST.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q449_Serialize_and_Deserialize_BST {
+}
Index: src/main/java/com/answer/graph/Q1254_Number_of_Closed_Islands.java
===================================================================
diff --git a/src/main/java/com/answer/graph/Q1254_Number_of_Closed_Islands.java b/src/main/java/com/answer/graph/Q1254_Number_of_Closed_Islands.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/graph/Q1254_Number_of_Closed_Islands.java	
@@ -0,0 +1,4 @@
+package com.answer.graph;
+
+public class Q1254_Number_of_Closed_Islands {
+}
Index: src/main/java/com/answer/tree/Q549_Binary_Tree_Longest_Consecutive_Sequence_II.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q549_Binary_Tree_Longest_Consecutive_Sequence_II.java b/src/main/java/com/answer/tree/Q549_Binary_Tree_Longest_Consecutive_Sequence_II.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q549_Binary_Tree_Longest_Consecutive_Sequence_II.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q549_Binary_Tree_Longest_Consecutive_Sequence_II {
+}
Index: src/main/java/com/answer/tree/Q589_N_ary_Tree_Preorder_Traversal.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q589_N_ary_Tree_Preorder_Traversal.java b/src/main/java/com/answer/tree/Q589_N_ary_Tree_Preorder_Traversal.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q589_N_ary_Tree_Preorder_Traversal.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q589_N_ary_Tree_Preorder_Traversal {
+}
Index: src/main/java/com/answer/graph/Q323_Number_of_Connected_Components_in_an_Undirected_Graph.java
===================================================================
diff --git a/src/main/java/com/answer/graph/Q323_Number_of_Connected_Components_in_an_Undirected_Graph.java b/src/main/java/com/answer/graph/Q323_Number_of_Connected_Components_in_an_Undirected_Graph.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/graph/Q323_Number_of_Connected_Components_in_an_Undirected_Graph.java	
@@ -0,0 +1,4 @@
+package com.answer.graph;
+
+public class Q323_Number_of_Connected_Components_in_an_Undirected_Graph {
+}
Index: src/main/java/com/answer/trie/Q648_Replace_Words.java
===================================================================
diff --git a/src/main/java/com/answer/trie/Q648_Replace_Words.java b/src/main/java/com/answer/trie/Q648_Replace_Words.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/trie/Q648_Replace_Words.java	
@@ -0,0 +1,4 @@
+package com.answer.trie;
+
+public class Q648_Replace_Words {
+}
Index: src/main/java/com/answer/graph/Q886_Possible_Bipartition.java
===================================================================
diff --git a/src/main/java/com/answer/graph/Q886_Possible_Bipartition.java b/src/main/java/com/answer/graph/Q886_Possible_Bipartition.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/graph/Q886_Possible_Bipartition.java	
@@ -0,0 +1,4 @@
+package com.answer.graph;
+
+public class Q886_Possible_Bipartition {
+}
Index: src/main/java/com/answer/trie/Q677_Map_Sum_Pairs.java
===================================================================
diff --git a/src/main/java/com/answer/trie/Q677_Map_Sum_Pairs.java b/src/main/java/com/answer/trie/Q677_Map_Sum_Pairs.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/trie/Q677_Map_Sum_Pairs.java	
@@ -0,0 +1,4 @@
+package com.answer.trie;
+
+public class Q677_Map_Sum_Pairs {
+}
Index: src/main/java/com/answer/tree/Q1008_Construct_Binary_Search_Tree_from_Preorder_Traversal.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q1008_Construct_Binary_Search_Tree_from_Preorder_Traversal.java b/src/main/java/com/answer/tree/Q1008_Construct_Binary_Search_Tree_from_Preorder_Traversal.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q1008_Construct_Binary_Search_Tree_from_Preorder_Traversal.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q1008_Construct_Binary_Search_Tree_from_Preorder_Traversal {
+}
Index: src/main/java/com/answer/graph/Q1020_Number_of_Enclaves.java
===================================================================
diff --git a/src/main/java/com/answer/graph/Q1020_Number_of_Enclaves.java b/src/main/java/com/answer/graph/Q1020_Number_of_Enclaves.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/graph/Q1020_Number_of_Enclaves.java	
@@ -0,0 +1,4 @@
+package com.answer.graph;
+
+public class Q1020_Number_of_Enclaves {
+}
Index: src/main/java/com/answer/graph/Q1034_Coloring_A_Border.java
===================================================================
diff --git a/src/main/java/com/answer/graph/Q1034_Coloring_A_Border.java b/src/main/java/com/answer/graph/Q1034_Coloring_A_Border.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/graph/Q1034_Coloring_A_Border.java	
@@ -0,0 +1,4 @@
+package com.answer.graph;
+
+public class Q1034_Coloring_A_Border {
+}
Index: src/main/java/com/answer/trie/Q642_Design_Search_Autocomplete_System.java
===================================================================
diff --git a/src/main/java/com/answer/trie/Q642_Design_Search_Autocomplete_System.java b/src/main/java/com/answer/trie/Q642_Design_Search_Autocomplete_System.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/trie/Q642_Design_Search_Autocomplete_System.java	
@@ -0,0 +1,4 @@
+package com.answer.trie;
+
+public class Q642_Design_Search_Autocomplete_System {
+}
Index: src/main/java/com/answer/graph/Q802_Find_Eventual_Safe_States.java
===================================================================
diff --git a/src/main/java/com/answer/graph/Q802_Find_Eventual_Safe_States.java b/src/main/java/com/answer/graph/Q802_Find_Eventual_Safe_States.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/graph/Q802_Find_Eventual_Safe_States.java	
@@ -0,0 +1,4 @@
+package com.answer.graph;
+
+public class Q802_Find_Eventual_Safe_States {
+}
Index: src/main/java/com/answer/topological_sort/Q207_Course_Schedule.java
===================================================================
diff --git a/src/main/java/com/answer/topological_sort/Q207_Course_Schedule.java b/src/main/java/com/answer/topological_sort/Q207_Course_Schedule.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/topological_sort/Q207_Course_Schedule.java	
@@ -0,0 +1,4 @@
+package com.answer.topological_sort;
+
+public class Q207_Course_Schedule {
+}
Index: src/main/java/com/answer/tree/Q426_Convert_Binary_Search_Tree_to_Sorted_Doubly_Linked_List.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q426_Convert_Binary_Search_Tree_to_Sorted_Doubly_Linked_List.java b/src/main/java/com/answer/tree/Q426_Convert_Binary_Search_Tree_to_Sorted_Doubly_Linked_List.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q426_Convert_Binary_Search_Tree_to_Sorted_Doubly_Linked_List.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q426_Convert_Binary_Search_Tree_to_Sorted_Doubly_Linked_List {
+}
Index: src/main/java/com/answer/topological_sort/Q210_Course_Schedule_II.java
===================================================================
diff --git a/src/main/java/com/answer/topological_sort/Q210_Course_Schedule_II.java b/src/main/java/com/answer/topological_sort/Q210_Course_Schedule_II.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/topological_sort/Q210_Course_Schedule_II.java	
@@ -0,0 +1,4 @@
+package com.answer.topological_sort;
+
+public class Q210_Course_Schedule_II {
+}
Index: src/main/java/com/answer/tree/Q95_Unique_Binary_Search_Trees_II.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q95_Unique_Binary_Search_Trees_II.java b/src/main/java/com/answer/tree/Q95_Unique_Binary_Search_Trees_II.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q95_Unique_Binary_Search_Trees_II.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q95_Unique_Binary_Search_Trees_II {
+}
Index: src/main/java/com/answer/linkedlist/Q138_Copy_List_with_Random_Pointer.java
===================================================================
diff --git a/src/main/java/com/answer/linkedlist/Q138_Copy_List_with_Random_Pointer.java b/src/main/java/com/answer/linkedlist/Q138_Copy_List_with_Random_Pointer.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/linkedlist/Q138_Copy_List_with_Random_Pointer.java	
@@ -0,0 +1,4 @@
+package com.answer.linkedlist;
+
+public class Q138_Copy_List_with_Random_Pointer {
+}
Index: src/main/java/com/answer/priorityqueue/Q1046_Last_Stone_Weight.java
===================================================================
diff --git a/src/main/java/com/answer/priorityqueue/Q1046_Last_Stone_Weight.java b/src/main/java/com/answer/priorityqueue/Q1046_Last_Stone_Weight.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/priorityqueue/Q1046_Last_Stone_Weight.java	
@@ -0,0 +1,4 @@
+package com.answer.priorityqueue;
+
+public class Q1046_Last_Stone_Weight {
+}
Index: src/main/java/com/answer/math/Q779_K_th_Symbol_in_Grammar.java
===================================================================
diff --git a/src/main/java/com/answer/math/Q779_K_th_Symbol_in_Grammar.java b/src/main/java/com/answer/math/Q779_K_th_Symbol_in_Grammar.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/math/Q779_K_th_Symbol_in_Grammar.java	
@@ -0,0 +1,4 @@
+package com.answer.math;
+
+public class Q779_K_th_Symbol_in_Grammar {
+}
Index: src/main/java/com/answer/priorityqueue/Q1167_Minimum_Cost_to_Connect_Sticks.java
===================================================================
diff --git a/src/main/java/com/answer/priorityqueue/Q1167_Minimum_Cost_to_Connect_Sticks.java b/src/main/java/com/answer/priorityqueue/Q1167_Minimum_Cost_to_Connect_Sticks.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/priorityqueue/Q1167_Minimum_Cost_to_Connect_Sticks.java	
@@ -0,0 +1,4 @@
+package com.answer.priorityqueue;
+
+public class Q1167_Minimum_Cost_to_Connect_Sticks {
+}
Index: src/main/java/com/answer/priorityqueue/Q1337_The_K_Weakest_Rows_in_a_Matrix.java
===================================================================
diff --git a/src/main/java/com/answer/priorityqueue/Q1337_The_K_Weakest_Rows_in_a_Matrix.java b/src/main/java/com/answer/priorityqueue/Q1337_The_K_Weakest_Rows_in_a_Matrix.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/priorityqueue/Q1337_The_K_Weakest_Rows_in_a_Matrix.java	
@@ -0,0 +1,4 @@
+package com.answer.priorityqueue;
+
+public class Q1337_The_K_Weakest_Rows_in_a_Matrix {
+}
Index: src/main/java/com/answer/priorityqueue/Q1642_Furthest_Building_You_Can_Reach.java
===================================================================
diff --git a/src/main/java/com/answer/priorityqueue/Q1642_Furthest_Building_You_Can_Reach.java b/src/main/java/com/answer/priorityqueue/Q1642_Furthest_Building_You_Can_Reach.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/priorityqueue/Q1642_Furthest_Building_You_Can_Reach.java	
@@ -0,0 +1,4 @@
+package com.answer.priorityqueue;
+
+public class Q1642_Furthest_Building_You_Can_Reach {
+}
Index: src/main/java/com/answer/array/Q1295_Find_Numbers_with_Even_Number_of_Digits.java
===================================================================
diff --git a/src/main/java/com/answer/array/Q1295_Find_Numbers_with_Even_Number_of_Digits.java b/src/main/java/com/answer/array/Q1295_Find_Numbers_with_Even_Number_of_Digits.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/array/Q1295_Find_Numbers_with_Even_Number_of_Digits.java	
@@ -0,0 +1,4 @@
+package com.answer.array;
+
+public class Q1295_Find_Numbers_with_Even_Number_of_Digits {
+}
Index: src/main/java/com/answer/union_find/Q1319_Number_of_Operations_to_Make_Network_Connected.java
===================================================================
diff --git a/src/main/java/com/answer/union_find/Q1319_Number_of_Operations_to_Make_Network_Connected.java b/src/main/java/com/answer/union_find/Q1319_Number_of_Operations_to_Make_Network_Connected.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/union_find/Q1319_Number_of_Operations_to_Make_Network_Connected.java	
@@ -0,0 +1,4 @@
+package com.answer.union_find;
+
+public class Q1319_Number_of_Operations_to_Make_Network_Connected {
+}
Index: src/main/java/com/answer/tree/Q1372_Longest_ZigZag_Path_in_a_Binary_Tree.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q1372_Longest_ZigZag_Path_in_a_Binary_Tree.java b/src/main/java/com/answer/tree/Q1372_Longest_ZigZag_Path_in_a_Binary_Tree.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q1372_Longest_ZigZag_Path_in_a_Binary_Tree.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q1372_Longest_ZigZag_Path_in_a_Binary_Tree {
+}
Index: src/main/java/com/answer/array/Q1089_Duplicate_Zeros.java
===================================================================
diff --git a/src/main/java/com/answer/array/Q1089_Duplicate_Zeros.java b/src/main/java/com/answer/array/Q1089_Duplicate_Zeros.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/array/Q1089_Duplicate_Zeros.java	
@@ -0,0 +1,4 @@
+package com.answer.array;
+
+public class Q1089_Duplicate_Zeros {
+}
Index: src/main/java/com/answer/union_find/Q959_Regions_Cut_By_Slashes.java
===================================================================
diff --git a/src/main/java/com/answer/union_find/Q959_Regions_Cut_By_Slashes.java b/src/main/java/com/answer/union_find/Q959_Regions_Cut_By_Slashes.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/union_find/Q959_Regions_Cut_By_Slashes.java	
@@ -0,0 +1,4 @@
+package com.answer.union_find;
+
+public class Q959_Regions_Cut_By_Slashes {
+}
Index: src/main/java/com/answer/tree/Q606_Construct_String_from_Binary_Tree.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q606_Construct_String_from_Binary_Tree.java b/src/main/java/com/answer/tree/Q606_Construct_String_from_Binary_Tree.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q606_Construct_String_from_Binary_Tree.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q606_Construct_String_from_Binary_Tree {
+}
Index: src/main/java/com/answer/union_find/Q1202_Smallest_String_With_Swaps.java
===================================================================
diff --git a/src/main/java/com/answer/union_find/Q1202_Smallest_String_With_Swaps.java b/src/main/java/com/answer/union_find/Q1202_Smallest_String_With_Swaps.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/union_find/Q1202_Smallest_String_With_Swaps.java	
@@ -0,0 +1,4 @@
+package com.answer.union_find;
+
+public class Q1202_Smallest_String_With_Swaps {
+}
Index: src/main/java/com/answer/tree/Q865_Smallest_Subtree_with_all_the_Deepest_Nodes.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q865_Smallest_Subtree_with_all_the_Deepest_Nodes.java b/src/main/java/com/answer/tree/Q865_Smallest_Subtree_with_all_the_Deepest_Nodes.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q865_Smallest_Subtree_with_all_the_Deepest_Nodes.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q865_Smallest_Subtree_with_all_the_Deepest_Nodes {
+}
Index: src/main/java/com/answer/dfs_bfs/Q126_Word_Ladder_II.java
===================================================================
diff --git a/src/main/java/com/answer/dfs_bfs/Q126_Word_Ladder_II.java b/src/main/java/com/answer/dfs_bfs/Q126_Word_Ladder_II.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/dfs_bfs/Q126_Word_Ladder_II.java	
@@ -0,0 +1,4 @@
+package com.answer.dfs_bfs;
+
+public class Q126_Word_Ladder_II {
+}
Index: src/main/java/com/answer/priorityqueue/Q973_K_Closest_Points_to_Origin.java
===================================================================
diff --git a/src/main/java/com/answer/priorityqueue/Q973_K_Closest_Points_to_Origin.java b/src/main/java/com/answer/priorityqueue/Q973_K_Closest_Points_to_Origin.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/priorityqueue/Q973_K_Closest_Points_to_Origin.java	
@@ -0,0 +1,4 @@
+package com.answer.priorityqueue;
+
+public class Q973_K_Closest_Points_to_Origin {
+}
Index: src/main/java/com/answer/tree/Q298_Binary_Tree_Longest_Consecutive_Sequence.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q298_Binary_Tree_Longest_Consecutive_Sequence.java b/src/main/java/com/answer/tree/Q298_Binary_Tree_Longest_Consecutive_Sequence.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q298_Binary_Tree_Longest_Consecutive_Sequence.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q298_Binary_Tree_Longest_Consecutive_Sequence {
+}
Index: src/main/java/com/answer/bit/Q89_Gray_Code.java
===================================================================
diff --git a/src/main/java/com/answer/bit/Q89_Gray_Code.java b/src/main/java/com/answer/bit/Q89_Gray_Code.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/bit/Q89_Gray_Code.java	
@@ -0,0 +1,4 @@
+package com.answer.bit;
+
+public class Q89_Gray_Code {
+}
Index: src/main/java/com/answer/tree/Q1102_Path_With_Maximum_Minimum_Value.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q1102_Path_With_Maximum_Minimum_Value.java b/src/main/java/com/answer/tree/Q1102_Path_With_Maximum_Minimum_Value.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q1102_Path_With_Maximum_Minimum_Value.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q1102_Path_With_Maximum_Minimum_Value {
+}
Index: src/main/java/com/answer/tree/Q331_Verify_Preorder_Serialization_of_a_Binary_Tree.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q331_Verify_Preorder_Serialization_of_a_Binary_Tree.java b/src/main/java/com/answer/tree/Q331_Verify_Preorder_Serialization_of_a_Binary_Tree.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q331_Verify_Preorder_Serialization_of_a_Binary_Tree.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q331_Verify_Preorder_Serialization_of_a_Binary_Tree {
+}
Index: src/main/java/com/answer/sorting/Q1200_Minimum_Absolute_Difference.java
===================================================================
diff --git a/src/main/java/com/answer/sorting/Q1200_Minimum_Absolute_Difference.java b/src/main/java/com/answer/sorting/Q1200_Minimum_Absolute_Difference.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/sorting/Q1200_Minimum_Absolute_Difference.java	
@@ -0,0 +1,4 @@
+package com.answer.sorting;
+
+public class Q1200_Minimum_Absolute_Difference {
+}
Index: src/main/java/com/answer/union_find/Q947_Most_Stones_Removed_with_Same_Row_or_Column.java
===================================================================
diff --git a/src/main/java/com/answer/union_find/Q947_Most_Stones_Removed_with_Same_Row_or_Column.java b/src/main/java/com/answer/union_find/Q947_Most_Stones_Removed_with_Same_Row_or_Column.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/union_find/Q947_Most_Stones_Removed_with_Same_Row_or_Column.java	
@@ -0,0 +1,4 @@
+package com.answer.union_find;
+
+public class Q947_Most_Stones_Removed_with_Same_Row_or_Column {
+}
Index: src/main/java/com/answer/tree/Q1367_Linked_List_in_Binary_Tree.java
===================================================================
diff --git a/src/main/java/com/answer/tree/Q1367_Linked_List_in_Binary_Tree.java b/src/main/java/com/answer/tree/Q1367_Linked_List_in_Binary_Tree.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/tree/Q1367_Linked_List_in_Binary_Tree.java	
@@ -0,0 +1,4 @@
+package com.answer.tree;
+
+public class Q1367_Linked_List_in_Binary_Tree {
+}
Index: src/main/java/com/answer/stack/Q542_01_Matrix.java
===================================================================
diff --git a/src/main/java/com/answer/stack/Q542_01_Matrix.java b/src/main/java/com/answer/stack/Q542_01_Matrix.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/stack/Q542_01_Matrix.java	
@@ -0,0 +1,4 @@
+package com.answer.stack;
+
+public class Q542_01_Matrix {
+}
Index: src/main/java/com/answer/stack/Q733_Flood_Fill.java
===================================================================
diff --git a/src/main/java/com/answer/stack/Q733_Flood_Fill.java b/src/main/java/com/answer/stack/Q733_Flood_Fill.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/stack/Q733_Flood_Fill.java	
@@ -0,0 +1,4 @@
+package com.answer.stack;
+
+public class Q733_Flood_Fill {
+}
Index: src/main/java/com/answer/backtracking/Q1593_Split_a_String_Into_the_Max_Number_of_Unique_Substrings.java
===================================================================
diff --git a/src/main/java/com/answer/backtracking/Q1593_Split_a_String_Into_the_Max_Number_of_Unique_Substrings.java b/src/main/java/com/answer/backtracking/Q1593_Split_a_String_Into_the_Max_Number_of_Unique_Substrings.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/backtracking/Q1593_Split_a_String_Into_the_Max_Number_of_Unique_Substrings.java	
@@ -0,0 +1,4 @@
+package com.answer.backtracking;
+
+public class Q1593_Split_a_String_Into_the_Max_Number_of_Unique_Substrings {
+}
Index: src/main/java/com/answer/backtracking/Q473_Matchsticks_to_Square.java
===================================================================
diff --git a/src/main/java/com/answer/backtracking/Q473_Matchsticks_to_Square.java b/src/main/java/com/answer/backtracking/Q473_Matchsticks_to_Square.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/backtracking/Q473_Matchsticks_to_Square.java	
@@ -0,0 +1,4 @@
+package com.answer.backtracking;
+
+public class Q473_Matchsticks_to_Square {
+}
Index: src/main/java/com/answer/stack/Q286_Walls_and_Gates.java
===================================================================
diff --git a/src/main/java/com/answer/stack/Q286_Walls_and_Gates.java b/src/main/java/com/answer/stack/Q286_Walls_and_Gates.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/stack/Q286_Walls_and_Gates.java	
@@ -0,0 +1,4 @@
+package com.answer.stack;
+
+public class Q286_Walls_and_Gates {
+}
Index: src/main/java/com/answer/linkedlist/Q430_Flatten_a_Multilevel_Doubly_Linked_List.java
===================================================================
diff --git a/src/main/java/com/answer/linkedlist/Q430_Flatten_a_Multilevel_Doubly_Linked_List.java b/src/main/java/com/answer/linkedlist/Q430_Flatten_a_Multilevel_Doubly_Linked_List.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/linkedlist/Q430_Flatten_a_Multilevel_Doubly_Linked_List.java	
@@ -0,0 +1,4 @@
+package com.answer.linkedlist;
+
+public class Q430_Flatten_a_Multilevel_Doubly_Linked_List {
+}
Index: src/main/java/com/answer/linkedlist/Q61_Rotate_List.java
===================================================================
diff --git a/src/main/java/com/answer/linkedlist/Q61_Rotate_List.java b/src/main/java/com/answer/linkedlist/Q61_Rotate_List.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/linkedlist/Q61_Rotate_List.java	
@@ -0,0 +1,4 @@
+package com.answer.linkedlist;
+
+public class Q61_Rotate_List {
+}
Index: src/main/java/com/answer/linkedlist/Q708_Insert_into_a_Sorted_Circular_Linked_List.java
===================================================================
diff --git a/src/main/java/com/answer/linkedlist/Q708_Insert_into_a_Sorted_Circular_Linked_List.java b/src/main/java/com/answer/linkedlist/Q708_Insert_into_a_Sorted_Circular_Linked_List.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/linkedlist/Q708_Insert_into_a_Sorted_Circular_Linked_List.java	
@@ -0,0 +1,4 @@
+package com.answer.linkedlist;
+
+public class Q708_Insert_into_a_Sorted_Circular_Linked_List {
+}
Index: src/main/java/com/answer/backtracking/Q1079_Letter_Tile_Possibilities.java
===================================================================
diff --git a/src/main/java/com/answer/backtracking/Q1079_Letter_Tile_Possibilities.java b/src/main/java/com/answer/backtracking/Q1079_Letter_Tile_Possibilities.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/backtracking/Q1079_Letter_Tile_Possibilities.java	
@@ -0,0 +1,4 @@
+package com.answer.backtracking;
+
+public class Q1079_Letter_Tile_Possibilities {
+}
Index: src/main/java/com/answer/hashmap/Q249_Group_Shifted_Strings.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q249_Group_Shifted_Strings.java b/src/main/java/com/answer/hashmap/Q249_Group_Shifted_Strings.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q249_Group_Shifted_Strings.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public class Q249_Group_Shifted_Strings {
+}
Index: src/main/java/com/answer/hashmap/Q288_Unique_Word_Abbreviation.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q288_Unique_Word_Abbreviation.java b/src/main/java/com/answer/hashmap/Q288_Unique_Word_Abbreviation.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q288_Unique_Word_Abbreviation.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public class Q288_Unique_Word_Abbreviation {
+}
Index: src/main/java/com/answer/hashmap/Q311_Sparse_Matrix_Multiplication.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q311_Sparse_Matrix_Multiplication.java b/src/main/java/com/answer/hashmap/Q311_Sparse_Matrix_Multiplication.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q311_Sparse_Matrix_Multiplication.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public class Q311_Sparse_Matrix_Multiplication {
+}
Index: src/main/java/com/answer/stack/Q752_Open_the_Lock.java
===================================================================
diff --git a/src/main/java/com/answer/stack/Q752_Open_the_Lock.java b/src/main/java/com/answer/stack/Q752_Open_the_Lock.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/stack/Q752_Open_the_Lock.java	
@@ -0,0 +1,4 @@
+package com.answer.stack;
+
+public class Q752_Open_the_Lock {
+}
Index: src/main/java/com/answer/hashmap/Q652_Find_Duplicate_Subtrees.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q652_Find_Duplicate_Subtrees.java b/src/main/java/com/answer/hashmap/Q652_Find_Duplicate_Subtrees.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q652_Find_Duplicate_Subtrees.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public interface Q652_Find_Duplicate_Subtrees {
+}
Index: src/main/java/com/answer/hashmap/Q771_Jewels_and_Stones.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q771_Jewels_and_Stones.java b/src/main/java/com/answer/hashmap/Q771_Jewels_and_Stones.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q771_Jewels_and_Stones.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public class Q771_Jewels_and_Stones {
+}
Index: src/main/java/com/answer/hashmap/Q359_Logger_Rate_Limiter.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q359_Logger_Rate_Limiter.java b/src/main/java/com/answer/hashmap/Q359_Logger_Rate_Limiter.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q359_Logger_Rate_Limiter.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public class Q359_Logger_Rate_Limiter {
+}
Index: src/main/java/com/answer/sorting/Q414_Third_Maximum_Number.java
===================================================================
diff --git a/src/main/java/com/answer/sorting/Q414_Third_Maximum_Number.java b/src/main/java/com/answer/sorting/Q414_Third_Maximum_Number.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/sorting/Q414_Third_Maximum_Number.java	
@@ -0,0 +1,4 @@
+package com.answer.sorting;
+
+public class Q414_Third_Maximum_Number {
+}
Index: src/main/java/com/answer/hashmap/Q36_Valid_Sudoku.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q36_Valid_Sudoku.java b/src/main/java/com/answer/hashmap/Q36_Valid_Sudoku.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q36_Valid_Sudoku.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public class Q36_Valid_Sudoku {
+}
Index: src/main/java/com/answer/trie/Q336_Palindrome_Pairs.java
===================================================================
diff --git a/src/main/java/com/answer/trie/Q336_Palindrome_Pairs.java b/src/main/java/com/answer/trie/Q336_Palindrome_Pairs.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/trie/Q336_Palindrome_Pairs.java	
@@ -0,0 +1,4 @@
+package com.answer.trie;
+
+public class Q336_Palindrome_Pairs {
+}
Index: src/main/java/com/answer/trie/Q421_Maximum_XOR_of_Two_Numbers_in_an_Array.java
===================================================================
diff --git a/src/main/java/com/answer/trie/Q421_Maximum_XOR_of_Two_Numbers_in_an_Array.java b/src/main/java/com/answer/trie/Q421_Maximum_XOR_of_Two_Numbers_in_an_Array.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/trie/Q421_Maximum_XOR_of_Two_Numbers_in_an_Array.java	
@@ -0,0 +1,4 @@
+package com.answer.trie;
+
+public class Q421_Maximum_XOR_of_Two_Numbers_in_an_Array {
+}
Index: src/main/java/com/answer/trie/Q425_Word_Squares.java
===================================================================
diff --git a/src/main/java/com/answer/trie/Q425_Word_Squares.java b/src/main/java/com/answer/trie/Q425_Word_Squares.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/trie/Q425_Word_Squares.java	
@@ -0,0 +1,4 @@
+package com.answer.trie;
+
+public class Q425_Word_Squares {
+}
Index: src/main/java/com/answer/hashmap/Q380_Insert_Delete_GetRandom_O1.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q380_Insert_Delete_GetRandom_O1.java b/src/main/java/com/answer/hashmap/Q380_Insert_Delete_GetRandom_O1.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q380_Insert_Delete_GetRandom_O1.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public class Q380_Insert_Delete_GetRandom_O1 {
+}
Index: src/main/java/com/answer/hashmap/Q599_Minimum_Index_Sum_of_Two_Lists.java
===================================================================
diff --git a/src/main/java/com/answer/hashmap/Q599_Minimum_Index_Sum_of_Two_Lists.java b/src/main/java/com/answer/hashmap/Q599_Minimum_Index_Sum_of_Two_Lists.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/hashmap/Q599_Minimum_Index_Sum_of_Two_Lists.java	
@@ -0,0 +1,4 @@
+package com.answer.hashmap;
+
+public class Q599_Minimum_Index_Sum_of_Two_Lists {
+}
Index: src/main/java/com/answer/trie/Q211_Design_Add_and_Search_Words_Data_Structure.java
===================================================================
diff --git a/src/main/java/com/answer/trie/Q211_Design_Add_and_Search_Words_Data_Structure.java b/src/main/java/com/answer/trie/Q211_Design_Add_and_Search_Words_Data_Structure.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/trie/Q211_Design_Add_and_Search_Words_Data_Structure.java	
@@ -0,0 +1,4 @@
+package com.answer.trie;
+
+public class Q211_Design_Add_and_Search_Words_Data_Structure {
+}
Index: src/main/java/com/answer/sorting/Q2343_Query_Kth_Smallest_Trimmed_Number.java
===================================================================
diff --git a/src/main/java/com/answer/sorting/Q2343_Query_Kth_Smallest_Trimmed_Number.java b/src/main/java/com/answer/sorting/Q2343_Query_Kth_Smallest_Trimmed_Number.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/com/answer/sorting/Q2343_Query_Kth_Smallest_Trimmed_Number.java	
@@ -0,0 +1,4 @@
+package com.answer.sorting;
+
+public class Q2343_Query_Kth_Smallest_Trimmed_Number {
+}
